addon/validators/presence.js

```diff
@@ -12,7 +12,6 @@ export default function validatePresence(options) {
     if (typeof result === 'boolean' || typeof result === 'string') {
       return result;
     } else {
-
       // We flipped the meaning behind `present` and `blank` so switch the two
       if (result.type === 'present') {
```

Comment:
No the actual error messages are coming back flip. Some of the tests are looking at the the changeset errors object and the messages are flipped.